corehq/apps/userreports/transforms/specs.py

```diff
@@ -75,3 +75,16 @@ def transform_function(value):
                 return value
 
         return transform_function
+
+
+class TranslationTransform(Transform):
+    type = TypeProperty('translation')
+    translations = DictProperty()
+
+    def get_transform_function(self):
+
+        # For now, use the identity function
```

Comment:
I think this comment should be more explicit that it's not supported yet. also wouldn't it be somewhat trivial to support? I think you can access the current language from a built in function (don't know how off the top of my head but should be documented) and then look it up in the translations dict